reference-architecture/vmware-ansible/roles/vmware-guest-setup/tasks/main.yaml

```diff
@@ -0,0 +1,20 @@
+---
+- name: be sure chrony is installed
+  yum: name=chrony state=installed
```

Comment:
For 3.3+ the openshift_clock role is enabled  by default, which will configure chrony/ntpd depending on if the host is an atomic host.